modules/nf-core/happy/happy/main.nf

```diff
@@ -9,29 +9,49 @@ process HAPPY_HAPPY {
         'quay.io/biocontainers/hap.py:0.3.14--py27h5c5a3ab_0' }"
 
     input:
-    tuple val(meta), path(truth_vcf), path(query_vcf), path(bed)
-    tuple path(fasta), path(fasta_fai)
+    tuple val(meta), path(query_vcf), path(truth_vcf), path(regions_bed), path(targets_bed)
+    path(fasta)
+    path(fasta_fai)
+    path(false_positives_bed)
+    path(stratification_tsv)
```

Comment:
I guess according with all other modules these ones should get metas too

Reply:
Good spot, thanks!